### PR TITLE
DDF-1834: Missing dependencies break parallel and make-dependents builds

### PR DESCRIPTION
**Pull request by lessarderic**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/codice/ddf/pull/502" target="_blank">ReviewNinja

File: catalog/catalog-app/pom.xml [hunk 1]
```diff
@@ -12,9 +12,7 @@
  *
  **/
 -->
-<project xmlns="http://maven.apache.org/POM/4.0.0"
-         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
```
Comment (brendan-hofmann) [inline catalog/catalog-app/pom.xml]: Looks like you just formatted the header here?

**Comment by brendan-hofmann** [comment]
LGTM :+1:

File: catalog/catalog-app/pom.xml (hunk 1 above)
Comment (lessarderic) [inline catalog/catalog-app/pom.xml]: The script did, yes. Not me :-)

Comment (oscaritoro) [inline catalog/catalog-app/pom.xml]: Also, looks like you went away from how our formatter does it (at least in my intelliJ)

Comment (lessarderic) [inline catalog/catalog-app/pom.xml]: Let me reformat the files and push again then.

**Comment by oscaritoro** [comment]
LGTM :+1:

**Pull request merged**